### PR TITLE
Mule 12653: Reduce performance build time and PS iterations.

### PR DESCRIPTION
**Pull request by LucianoGandini**
Changes added makes performance builds successful (https://muleesb.ci.cloudbees.com/view/Mule%204/job/Mule-CE-Performance-Runner/31/).

### CODE REVIEW

File: tests/performance/src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <AsyncLogger name="org.apache" level="ERROR"/>
+
+        <!-- Reduce startup noise -->
+        <AsyncLogger name="org.springframework.beans.factory" level="WARN"/>
+
+        <!-- Mule classes -->
+        <AsyncLogger name="org.mule" level="WARN"/>
+        <AsyncLogger name="com.mulesoft" level="WARN"/>
+
+        <AsyncRoot level="INFO">
```
Comment (dfeist) [inline tests/performance/src/main/resources/log4j2.xml]: Do we need info?

File: tests/performance/src/main/java/org/mule/FlowNullProcessorBenchmark.java [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
   @Override
   protected int getStreamIterations() {
-    return 1000;
+    return 100;
```
Comment (dfeist) [inline tests/performance/src/main/java/org/mule/FlowNullProcessorBenchmark.java]: Leave iterations at 1000 if problem strategy is commented out anyway

Comment (LucianoGandini) [inline tests/performance/src/main/java/org/mule/FlowNullProcessorBenchmark.java]: Result numbers were really high and you mentioned that JMH has issues with it.

File: tests/performance/src/main/resources/log4j2.xml (hunk 1 above)
Comment (LucianoGandini) [inline tests/performance/src/main/resources/log4j2.xml]: It replicates test configuration.

**Pull request merged**